### PR TITLE
June 26th prod release

### PR DESCRIPTION
**Pull request by richtier**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +29,9 @@
 - XOT-939 Fixed camel case URLs redirection for UKEF pages
 - No ticket - upgrade django to fix vulnerability
 
+- XOT-939 Fixed camel case URLs redirection for UKEF pages
```
Comment (sdonk) [inline CHANGELOG.md]: Uhm?

File: makefile [hunk 2]
```diff
@@ -75,6 +75,7 @@ DEBUG_SET_ENV_VARS := \
 	export UKEF_FORM_SUBMIT_TRACKER_URL=http://go.pardot.com/l/590031/2018-08-16/5kj25l; \
 	export UKEF_CONTACT_AGENT_EMAIL_ADDRESS=ukef_contact@example.com; \
 	export DIRECTORY_FORMS_API_BASE_URL=http://forms.trade.great:8011; \
+	export DIRECTORY_FORMS_DEFAULT_TIMEOUT=30; \
```
Comment (sdonk) [inline makefile]: Do we need this in vault?

File: CHANGELOG.md (hunk 1 above)
Comment (richtier) [reply CHANGELOG.md]: I confirmed that was indeed in the previous release :man_shrugging:

File: makefile (hunk 2 above)
Comment (richtier) [reply makefile]: nope - the current default timeout is ok in prod :)

**Pull request merged**